deployment.rst

```diff
@@ -257,3 +254,7 @@ Learn More
 .. _`Symfony plugin`: https://github.com/capistrano/symfony/
 .. _`Deployer`: http://deployer.org/
 .. _`Git Tagging`: https://git-scm.com/book/en/v2/Git-Basics-Tagging
+.. _`Heroku`: https://devcenter.heroku.com/articles/getting-started-with-symfony
+.. _`platform.sh`: https://docs.platform.sh/frameworks/symfony.html
+.. _`Azure`: https://azure.microsoft.com/en-us/develop/php/
+.. _`fortrabbit`: https://help.fortrabbit.com/install-symfony-3-uni
```

Comment:
Please use https://help.fortrabbit.com/install-symfony
This way we are not tied to the version.

Reply:
Absolutely - much nicer link :)